src/modules/uORB/uORBDeviceNode.cpp

```diff
@@ -44,12 +44,54 @@
 
 static uORB::SubscriptionInterval *filp_to_subscription(cdev::file_t *filp) { return static_cast<uORB::SubscriptionInterval *>(filp->f_priv); }
 
+// Determine the data range
+template <typename T>
+static inline bool is_in_range(const T &left, const T &value, const T &right)
```

Comment:
Why is this a template?

Reply:
I originally thought that the `_generation` might change the type, but this is not necessary. I will delete the template later for clarity.